packages/react/src/components/Button/Button.tsx

```diff
@@ -77,6 +77,9 @@ export interface ButtonProps
   /** A button can emphasize that it represents an alternative action. */
   secondary?: boolean
 
+  /** Inverted buttons are meant to be shown on dark backgrounds */
+  inverted?: boolean
```

Comment:
I would call this something else, because `inverted` is being used for `Input` and `Dropdown` with a very specific set of background colors.

Example) For `inverted` Input and Dropdown it's swapping the background colors
![image](https://user-images.githubusercontent.com/828692/67890265-a4d3c780-fb0d-11e9-9fa0-a773bfbf11bc.png)

And this button is a different kind of variant than that.